samples/matter/lock/Kconfig.sysbuild

```diff
@@ -11,7 +11,7 @@ config NETCORE_IPC_RADIO_IEEE802154
 	default y if BOARD_NRF5340DK_NRF5340_CPUAPP
 
 config NETCORE_IPC_RADIO_BT_HCI_IPC
-	default y if (BOARD_NRF7002DK_NRF5340_CPUAPP || BOARD_NRF5340DK_NRF5340_CPUAPP)
+	default y if (BOARD_NRF7002DK_NRF5340_CPUAPP || BOARD_NRF7002DK_NRF5340_CPUAPP_NRF7001 || BOARD_NRF5340DK_NRF5340_CPUAPP)
```

Comment:
This is only a part of the fix. There is still missing the `nrf7002dk_nrf5340_cpuapp_nrf7001.conf` file in `samples/matter/lock/sysbuild/ipc_radio/boards`. This file should be the same as `nrf7002dk_nrf5340_cpuapp.conf` file.